commit ba1d12f43e38eaa14c555fe48e8361a9e958f4d7 (refs/changes/12/1612/2)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2017-03-07T11:09:32+01:00 (2 years, 7 months ago)
    
    TOOLS-1712 sdcadm shouldn't fail to update if there's no manta zone
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/lib/procedures/update-stateless-services-v1.js b/lib/procedures/update-stateless-services-v1.js
index 3365168..754375a 100644
--- a/lib/procedures/update-stateless-services-v1.js
+++ b/lib/procedures/update-stateless-services-v1.js
@@ -96,7 +96,7 @@ UpdateStatelessServices.prototype.execute = function ussv2Execute(opts, cb) {
         };
 
         if (!change.insts) {
-            change.insts = [change.inst];
+            change.insts = change.inst ? [change.inst] : [];
         }
 
         if (opts.plan.changes.length > 1) {
